setup.py

```diff
@@ -4,7 +4,7 @@
 __version__ = '5.0.5'
 
 requirements = [
-    "reportportal-client>=5.0.5",
+    "reportportal-client==5.0.5",
```

Comment:
Ok. Leave it as is for now. I'll make a fix for that shortly.

Reply:
You can now rebase against the master branch. It includes fixes compatible with the latest client version.